lib/collection/src/shard/conversions.rs

```diff
@@ -150,24 +150,44 @@ pub fn internal_create_index(
     shard: &RemoteShard,
     wait: bool,
 ) -> CreateFieldIndexCollectionInternal {
-    CreateFieldIndexCollectionInternal {
-        shard_id: shard.id,
-        create_field_index_collection: Some(CreateFieldIndexCollection {
-            collection_name: shard.collection_id.clone(),
-            wait: Some(wait),
-            field_name: create_index.field_name,
-            field_type: create_index.field_type.map(|field_type| match field_type {
+    let (field_type, field_index_params) = create_index
+        .field_type
+        .map(|field_schema| match field_schema {
```

Comment:
Each `field_schema` does not contain index params here. So that's why I propose change `None` position from
`
match field_schema {
  Keywork => (Keyword, None),
...
}
`
to
`
(match field_schema {
  Keywork => Keyword,
...
}, None)
`